Session/Calls/Call Management/SessionCallManager.swift

```diff
@@ -238,7 +232,7 @@ public final class SessionCallManager: NSObject, CallManagerProtocol {
                 else { return }
                 
                 if
-                    let conversationVC: ConversationVC = (presentingVC as? TopBannerController)?.wrappedViewController() as? ConversationVC,
+                    let conversationVC: ConversationVC = presentingVC as? ConversationVC,
```

Comment:
Should this be using something similar to `dependencies[singleton: .appContext].frontMostViewController`? (just wondering if there is a situation where the view hierarchy could result in this not returning the correct view controller)

Reply:
Hmmm... The `presentingVC` is actually the `dependencies[singleton: .appContext].frontMostViewController`, I'll change the name to make it less confusing.